### DISCUSSION TITLE
The alignment options don't work on the new text effect

### DISCUSSION THREAD

**Discussion by DarrenTAnims** [opentoonz/opentoonz#6501] [Ideas]
When you first add a text effect node, the text is shown centrally, but the alignment option shows "Left". And whatever you change the drop down to, it remains in the centre. I've tried refreshing the viewer and turning the preview mode on and off, none of which make any difference.

### Screenshot or Video Reference
![screencapture](https://user-images.githubusercontent.com/28575907/50738386-2ba86400-11cb-11e9-94bc-b36c99abe6f9.gif)

### OpenToonz Version
I'm using a nightly build from 14/12/18

**Comment by RodneyBaker** [comment]
I believe this is (unfortunately) just a known refresh issue.
Doing anything that refreshes the preview such as saving the Scene via Save All should update the canvas/preview.

**Comment by shun-iwasawa** [comment]
The refresh issue occurs only when you use note column as the text source.
For now the alignment option takes effect only when the text is rendered in multiple lines.

**Comment by artisteacher** [comment]
I think it would be great for the alignment options to extend to a single line - it's definitely the expected behavior.

**Comment by RodneyBaker** [comment]
Still an issue as of this date.  
My apology for mislabeling.
